internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -144,6 +145,12 @@ func resourceLogAnalyticsWorkspace() *pluginsdk.Resource {
 				ValidateFunc:     validation.FloatAtLeast(-1.0),
 			},
 
+			"default_data_collection_rule_id": {
```

Comment:
default is implied here since there isn't any other data collection rule ID property so can we rename this to 
```suggestion
			"data_collection_rule_id": {
```